RiotSharp.Test/RiotApiTest.cs

```diff
@@ -268,9 +268,9 @@ public void GetLeagues_BySummoner_Test()
         {
             EnsureCredibility(() =>
             {
-                var leagues = api.GetLeagues(RiotApiTestBase.summonersRegion, RiotApiTestBase.summonerIds);
+                var leagues = api.GetLeagues(RiotApiTestBase.summonersRegion, RiotApiTestBase.summonerIds.FirstOrDefault());
 
-                Assert.AreEqual(RiotApiTestBase.summonerIds.Distinct().Count(), leagues.Count);
+                Assert.IsTrue(leagues.Count > 0);
```

Comment:
It's better to use `Assert.NotEmpty(leagues)` I think. If this test fails, there's a better chance of getting a clearer failure message than for an arbitrary boolean condition.

Reply:
Yeah I discovered this some hours after my commit so I'm going to change this. Thanks TAGC.

Reply:
It's not Xunit :P 
But It would maybe a good idea to change to Xunit instead of MSTest.
What do you think @BenFradet ?

Reply:
In the longer term, why not :+1:

Reply:
Actually, I already thought about it, when I was migrating the project to .NET Standard. I would to do that when switching to deterministic unit testing.